### PR TITLE
added initial screen for left hand navigation 

### PR DESCRIPTION
**Pull request by hc-jordenlowe**
I have created a left handed nav with sub items.  

Tried the cashmere list, but it felt like it was restrictive on multiline items.  You had to explicited label each line.  If this gets recursive, this may be a big issue.  Of course if there is some advice, let me know.  Link to the list:  https://cashmere.healthcatalyst.net/components/list

Also, this is just a shell.  I dont have the service to call the API yet.  I just wanted to make sure the general look was acceptable before proceeding.

### CODE REVIEW

File: Fabric.Authorization.API/Modules/GrainsModule.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            Get("/",
+                async _ => await GetGrain().ConfigureAwait(false),
+                null,
+                "GetGrain");
+        }
+
+        private async Task<dynamic> GetGrain()
+        {
+            CheckReadAccess();
+            this.RequiresClaims(AuthorizationReadClaim);
```
Comment (michaelvidal24) [inline Fabric.Authorization.API/Modules/GrainsModule.cs]: The CheckReadAccess() function is a helper function that does what this.RequireClaims(AuthorizationReadClaim) does, we only need the CheckReadAccess() statement.

File: Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs [hunk 2]
```diff
@@ -38,6 +38,28 @@ public class SqlServerGrainStore : SqlServerBaseStore, IGrainStore
...
+        {
+            var grains = AuthorizationDbContext.Grains
+                .Include(g => g.SecurableItems)
+                .Where(g => !g.IsDeleted);
+
+            foreach (var grain in grains)
+            {
+                foreach (var securableItem in grain.SecurableItems)
+                {
+                    LoadChildrenRecursive(securableItem);
```
Comment (michaelvidal24) [inline Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: We may need to update this function to filter our deleted securables.

File: Fabric.Authorization.API/Modules/GrainsModule.cs (hunk 1 above)
Comment (hc-jordenlowe) [reply Fabric.Authorization.API/Modules/GrainsModule.cs]: gotcha.  Good to know

**Comment by hc-jordenlowe** [comment]
I have rebased.  I will add the feature flag.  Then the last thing will be to add tests and this section should be done.  Next bits on the task list:

1. Wire up the left pane with the right, this includes the highlighting
2. add expand/contract buttons for the lists of grains/securable items
3. add deep linking
4. see if we can migrate to models project

File: Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs (hunk 2 above)
Comment (hc-jordenlowe) [reply Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: This function is used within our securableItems code as well.  I would need to create something that is a separate function.

Comment (bsmithvt) [reply Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: Perhaps that's a bug in the existing code? Is there ever a case where we need to return deleted ones?

Comment (michaelvidal24) [reply Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: That is what I was thinking, we can just look to see what is using this function and determine if the current usages need deleted items or not.

Comment (bsmithvt) [reply Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: Tracing up the call stack, it's used in `GetRequiredWriteClaims(GrainService grainService, string grain)` in `FabricModule` and when adding a client. I'm thinking they *should* be filtered out for these scenarios.

Comment (michaelvidal24) [reply Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: Agreed.

Comment (hc-jordenlowe) [reply Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs]: kk, i will filter deleted stuff.  I went back to look for the other usage, and i cant find it.  Thought Securable items was working the same way. 

Ah well, added this code and if i run into this for securable items, i will change it.

**Pull request merged**